src/test_utils.rs

```diff
@@ -34,16 +34,18 @@ pub fn get_file_content<P: AsRef<Path>>(path: P) -> Vec<String> {
     }
     vec
 }
-pub fn get_lines_content<P: AsRef<Path>>(path: P) -> BTreeSet<String> {
+pub fn get_lines_content<P: AsRef<Path>>(path: P) -> Vec<String> {
```

Comment:
Inserting in a `BTreeSet` might be more efficient than pushing to a `Vec` and sorting only at the very end. On top of that, this change of code will now keep the duplicates: is it expected?

Reply:
Yes, we will keep the duplicates for tests comparison.

Reply:
I'm not sure that I understand the need for the changes in this file? I removed the changes on my machine and am still able to run the entire test suite without any problem.

Reply:
The problem is not visible in TransitModel tests, but is in a TartareTools test (see ND-1375).
This modification is very relevant.

Reply:
Yes, as discussed together, it is not directly related to the new feature being exposed in this PR, but it is still a bug in our test toolkit that was hiding some errors. So good for me!